helm/cloudcore/templates/configmap.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "cloudcore.name" . }}
+  namespace: kubeedge
+  labels:
+    {{- include "cloudcore.labels" . | nindent 4 }}
+data:
+  cloudcore.yaml: |
+    apiVersion: cloudcore.config.kubeedge.io/v1alpha1
+    kind: CloudCore
+    kubeAPIConfig:
+      kubeConfig: ""
+      master: ""
+    modules:
+      cloudHub:
+        nodeLimit: 10
```

Comment:
Should the configmap also include the `modules.cloudHub.https` block to expose the service running in `10002`?